react/src/hooks/projects/useProjects.ts

```diff
@@ -36,30 +31,40 @@ export const useProject = ({
   });
   return query;
 };
-export const useDsProjects =
-  (): UseQueryResult<DesignSafeProjectCollection> => {
-    const query = useGet<DesignSafeProjectCollection>({
-      endpoint: `/api/projects/v2/`,
-      key: ['projectsv2'],
-      apiService: ApiService.DesignSafe,
-    });
-    return query;
-  };
+export const useDsProjects = (): UseQueryResult<
+  DesignSafeProjectCollection | undefined
+> => {
+  const query = useGet<DesignSafeProjectCollection | undefined>({
+    endpoint: `/api/projects/v2/`,
+    key: ['projectsv2'],
+    apiService: ApiService.DesignSafe,
+  });
+  return query;
+};
 
 export const mergeDesignSafeProject = (
```

Comment:
* Could useProjects be used like useDsProject within mergeDesignSafeProject to reduce complexity when using the hook in components?
* follow on suggestion:  could this be renamed to something like useProjectsWithDesignSafeInformation 🤔 
* there is an issue currently where ProjectListing component is rendering table even though data isn't completely loaded and processed (will move to slack to discuss).

Reply:
Thanks for the suggestion! That was exactly what I was going for but couldn't get right. Appreciate the feedback and have updated the PR.